src/Util/Image.cpp

```diff
@@ -30,7 +30,7 @@ Image::Image(const std::string &filepath)
         };
 
     if (surface == nullptr) {
-        surface = {GetMissingTextureSDLSurface(), SDL_FreeSurface};
+        surface = {GetMissingImageTextureSDLSurface(), SDL_FreeSurface};
         LOG_ERROR("Failed to load image: '{}'", filepath);
         LOG_ERROR("{}", IMG_GetError());
```

Comment:
Even if your functions won't throw, I think it would be better to log the error first.

Reply:
Hey I already do it :(